Controller/Adminhtml/Import/Ajax.php

```diff
@@ -52,7 +53,11 @@ private function serializeRecords($records, $serializer) {
         $serializedData = [];
         
         foreach($records as $record) {
-            $serializedData[] = $serializer->serialize($record);
+            if($serializer->serialize($record)) {
+                $serializedData[] = $serializer->serialize($record);
```

Comment:
Why do you call `$serializer->serialize($record)` twice? Wouldn't that be unnecessarily expensive?